README.md

```diff
@@ -94,6 +94,49 @@ See [LICENSE](LICENSE).
 
 If you want to help out, see [CONTRIBUTING](CONTRIBUTING.md) for a set of guidelines.
 
+## Guidelines
+- We aim for complete C++11/14 compliance; please use this to your advantage
```

Comment:
I don't think monero aims for compliance. That would be nice though.

Reply:
C++11, from the makefiles.

Reply:
Aside from any static reference `new`'s,  a quick search proves otherwise:

```bash
$ grep -R " delete " src/|grep ";"|grep -vE "throw|LOG"
src/debug_utilities/cn_deserialize.cpp:    delete raw_data_file;
src/wallet/wallet_rpc_server.cpp:      delete m_wallet;
src/wallet/wallet_rpc_server.cpp:      delete m_wallet;
src/wallet/wallet_rpc_server.cpp:      er.message = "Failed to delete address book entry";
src/wallet/wallet_rpc_server.cpp:      delete m_wallet;
src/wallet/wallet_rpc_server.cpp:      delete m_wallet;
src/wallet/api/wallet_manager.cpp:        delete wallet_;
src/wallet/api/subaddress_account.cpp:     delete r;
src/wallet/api/transaction_history.cpp:        delete t;
src/wallet/api/transaction_history.cpp:    // delete old transactions;
src/wallet/api/transaction_history.cpp:        delete t;
src/wallet/api/wallet.cpp:    delete m_wallet2Callback;
src/wallet/api/wallet.cpp:    delete m_history;
src/wallet/api/wallet.cpp:    delete m_addressBook;
src/wallet/api/wallet.cpp:    delete m_subaddress;
src/wallet/api/wallet.cpp:    delete m_subaddressAccount;
src/wallet/api/wallet.cpp:    delete m_wallet;
src/wallet/api/wallet.cpp:    delete t;
src/wallet/api/subaddress.cpp:     delete r;
src/wallet/api/address_book.cpp:     delete r;
src/device/device_ledger.hpp:        device_ledger(const device_ledger &device) = delete ;
src/blockchain_utilities/bootstrap_file.cpp:  delete m_output_stream;
src/blockchain_utilities/bootstrap_file.cpp:  delete m_output_stream;
src/blockchain_utilities/bootstrap_file.cpp:  delete m_raw_data_file;
src/blockchain_utilities/blocksdat_file.cpp:  delete m_raw_data_file;
src/rpc/daemon_handler.cpp:      delete resp_message;
src/rpc/daemon_handler.cpp:        delete resp_message;
src/daemon/rpc_command_executor.cpp:    delete m_rpc_client;
src/cryptonote_core/blockchain.cpp:    delete db;
src/cryptonote_core/blockchain.cpp:  delete m_hardfork;
src/cryptonote_core/blockchain.cpp:  delete m_db;
src/blockchain_db/berkeleydb/db_bdb.cpp:    delete stats;
src/blockchain_db/berkeleydb/db_bdb.cpp:  delete db_stat1;
src/blockchain_db/berkeleydb/db_bdb.cpp:  delete db_stat2;
src/blockchain_db/lmdb/db_lmdb.cpp:    MFATAL("Please delete the existing database and resync.");
src/blockchain_db/lmdb/db_lmdb.cpp:    delete m_write_batch_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:  delete m_write_batch_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:  delete m_write_batch_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:  delete m_write_batch_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:      delete m_write_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:      delete m_write_txn;
src/blockchain_db/lmdb/db_lmdb.cpp:      delete m_write_txn;
src/p2p/net_node.inl:    MDEBUG("Attempting to delete IGD port mapping.");
src/common/perf_timer.cpp:    delete performance_timers;
src/common/dns_utils.cpp:    delete m_data;
```